src/coreclr/src/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -449,6 +449,43 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 }
                 break;
 
+            case NI_Vector64_CreateScalarUnsafe:
+            case NI_Vector128_CreateScalarUnsafe:
+                assert(isRMW);
+
+                if (intrin.op1->isContainedFltOrDblImmed())
+                {
+                    // fmov reg, #imm8
+                    const double dataValue = intrin.op1->AsDblCon()->gtDconVal;
+                    GetEmitter()->emitIns_R_F(ins, emitTypeSize(intrin.baseType), targetReg, dataValue, INS_OPTS_NONE);
+                }
+                else if (varTypeIsFloating(intrin.baseType))
+                {
+                    if (targetReg != op1Reg)
+                    {
+                        // fmov reg1, reg2
+                        GetEmitter()->emitIns_R_R(ins, emitTypeSize(intrin.baseType), targetReg, op1Reg, INS_OPTS_NONE);
+                    }
+                }
+                else
+                {
+                    assert(targetReg != op1Reg);
```

Comment:
This assert should be under else branch below - when op1 is contained op1Reg is REG_NA. However, I don't think you need this assert at all - since this block covers integer types of op1 and op1Reg is never a SIMD register (this will be asserted in emitIns_R_R_I anyway)